_rules/image-button-has-name.md

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
Rogue newline?

Reply:
If I change it, I am not allowed to commit just that change. Is this something we can live with?